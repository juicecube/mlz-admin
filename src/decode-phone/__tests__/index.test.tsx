import React from 'react';
import { testMount, testSnapshot } from '../../../tests';
import DecodePhone from '..';
import { mount } from 'enzyme';

describe('ðŸ§ª DecodePhone', () => {
  testMount(DecodePhone);
  testSnapshot(DecodePhone);

  it('Snapshot', () => {
    const wrapper = mount(
      <DecodePhone params="jZgOvsexafxJUlU3WHaMfA==">
        <span id="phone">155****1234</span>
      </DecodePhone>,
    );
    const div = wrapper.find('#phone').at(0);
    div.simulate('click');
    expect(wrapper.find('.ant-tooltip-inner').getDOMNode().innerHTML).not.toEqual(/åŠ è½½ä¸­/);
  });
});
