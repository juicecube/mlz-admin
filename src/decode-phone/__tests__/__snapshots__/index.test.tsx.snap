// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª DecodePhone snapshot test snapshot comparing 1`] = `"<span></span>"`;
